src/stories/ModalNext.stories.js

```diff
@@ -183,7 +183,8 @@ export const AlertWithActionButtons = {
     confirmButtonType: undefined,
     actionPrimaryLabel: 'Confirm',
     actionSecondaryLabel: 'Cancel',
-    actionPrimaryButtonType: undefined,
     showCloseButton: true,
+    onClickActionSecondary: () => console.log('secondary click'),
+    onClickActionPrimary: () => console.log('primary click'),
```

Comment:
Instead `console.log`, you can use the `action`, function of `@storybook/addon-actions` to improve the story experience.